src/patternfly/demos/Card/card-view-demo-template-gallery.hbs

```diff
@@ -180,4 +180,17 @@
         Expose REST services and their APIs using Swagger specification.
       {{/card-body}}
     {{/card}}
+    {{#> card card--id="card-11" card--modifier="pf-m-compact pf-l-bullseye"}}
```

Comment:
@mmenestr should this card be hoverable like the others?

Reply:
Hmm I want to say yes, to keep the card interaction the same as usual? But are the other cards hoverable with the idea that they're clickable, or is it just an added visual indication for the user to know where their cursor is?

Reply:
@mmenestr I'm assuming the latter since they aren't selectable or anything like that. The react demo is interactive, and it just seems like a visual enhancement - https://patternfly-react.surge.sh/components/card/react-demos/card-view/

Reply:
In that case I think let's just keep it consistent!